server/cmwell-ws/src/pack/logback.xml

```diff
@@ -24,6 +24,13 @@
         <encoder>
             <pattern>%date{ISO8601} %-44.44([%thread]) %-5level %-36.36logger{36} - %msg%n</pattern>
         </encoder>
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
```

Comment:
Please move this filter to the async appender. It will save context switches and won't put the log message in the async queue for nothing.